packer/install-dependencies.sh

```diff
@@ -104,6 +99,20 @@ for DASHBOARD_VERSION in ${DASHBOARD_VERSIONS}; do
     echo "  - ${CONTAINER_IMAGE}" >> ${VHD_LOGS_FILEPATH}
 done
 
+NEW_DASHBOARD_VERSIONS="2.0.0-beta8"
+for DASHBOARD_VERSION in ${NEW_DASHBOARD_VERSIONS}; do
+    CONTAINER_IMAGE="mcr.microsoft.com/oss/kubernetes/kubernetesui/dashboard:v${DASHBOARD_VERSION}"
+    pullContainerImage "docker" ${CONTAINER_IMAGE}
+    echo "  - ${CONTAINER_IMAGE}" >> ${VHD_LOGS_FILEPATH}
+done
+
+NEW_DASHBOARD_METRICS_SGRAPER_VERSIONS="1.0.2"
```

Comment:
docker pull mcr.microsoft.com/oss/kubernetes/kubernetesui/dashboard:v2.0.0-beta
Error response from daemon: manifest for mcr.microsoft.com/oss/kubernetes/kubernetesui/dashboard:v2.0.0-beta not found: manifest unknown: manifest unknown
tony@tonyxu-HP-Z240:~/MyWork/go/src/github.com/Azure/agentbaker$ docker pull mcr.microsoft.com/oss/kubernetes/kubernetesui/metrics-scraper:v1.0.2
Error response from daemon: manifest for mcr.microsoft.com/oss/kubernetes/kubernetesui/metrics-scraper:v1.0.2 not found: manifest unknown: manifest unknown